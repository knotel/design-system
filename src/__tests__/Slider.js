import React from 'react'
import renderer from 'react-test-renderer'
import { Slider } from '..'

describe('Slider', () => {
  test('renders', () => {
    const json = renderer
      .create(<Slider />, { disableLifecycleMethods: true })
      .toJSON()
    expect(json).toMatchSnapshot()
  })
})
