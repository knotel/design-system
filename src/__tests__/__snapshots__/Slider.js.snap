// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Slider renders 1`] = `
<div
  className="rc-slider kXenPj"
  onBlur={[Function]}
  onFocus={[Function]}
  onKeyDown={[Function]}
  onMouseDown={[Function]}
  onMouseUp={[Function]}
  onTouchStart={[Function]}
>
  <div
    className="rc-slider-rail"
    style={Object {}}
  />
  <div
    className="rc-slider-track rc-slider-track-1"
    style={
      Object {
        "left": "0%",
        "width": "0%",
      }
    }
  />
  <div
    className="rc-slider-step"
  />
  <div
    aria-disabled={false}
    aria-valuemax={100}
    aria-valuemin={0}
    aria-valuenow={0}
    className="rc-slider-handle rc-slider-handle-1"
    onBlur={[Function]}
    onKeyDown={[Function]}
    role="slider"
    style={
      Object {
        "left": "0%",
      }
    }
    tabIndex={0}
  />
  <div
    aria-disabled={false}
    aria-valuemax={100}
    aria-valuemin={0}
    aria-valuenow={0}
    className="rc-slider-handle rc-slider-handle-2"
    onBlur={[Function]}
    onKeyDown={[Function]}
    role="slider"
    style={
      Object {
        "left": "0%",
      }
    }
    tabIndex={0}
  />
  <div
    className="rc-slider-mark"
  />
</div>
`;
